validation/validation.py

```diff
@@ -64,17 +68,34 @@ def _check_prediction_values(df):
             errors.append(f"Column {PREDICTED_DAILY_NEW_CASES} contains NaN values")
         if any(df[PREDICTED_DAILY_NEW_CASES] < 0):
             errors.append(f"Column {PREDICTED_DAILY_NEW_CASES} contains negative values")
-        return errors
-    return None
+    return errors
 
 
-def _check_days(start_date, end_date, df):
+def _check_geos(ip_df, pred_df):
     errors = []
+    _add_geoid_column(ip_df)
+    _add_geoid_column(pred_df)
+    requested_geo_ids = set(ip_df.GeoID.unique())
+    actual_geo_ids = set(pred_df.GeoID.unique())
+    # Check if any missing
+    # Additional geos are OK, but predictions should at least include requested ones
```

Comment:
Something should be added to the ui to plot any additional countries with no ground truth. future work.

Reply:
@babakatwork requested this for the npi-demo as well -- somehow flag countries that were dropped for whatever reason, maybe with a grayed-out dot or something like that.